lang/de_de.lang

```diff
@@ -33,7 +33,7 @@ last_action_time = Letzte Aktion
 login_failed = Fehlgeschlagene Logins
 login_tries = Login Fehlversuche
 
-ycom_login_tries_info = Ab 10 Fehlversuchen kann sich der User nicht mehr anmelden.
+ycom_login_tries_info = Bitte die maximal erlaubten Fehlversuche in den Einstellungen beachten.
```

Comment:
nice to have: hier mit platzhalter arbeiten und den eingestellten wert in der meldung anzeigen